sympy/integrals/tests/test_integrals.py

```diff
@@ -1070,17 +1070,36 @@ def test_issue_2708():
 
 def test_issue_8368():
     assert integrate(exp(-s*x)*cosh(x), (x, 0, oo)) == \
-        Piecewise((pi*Piecewise((-s/(pi*(-s**2 + 1)), Abs(s**2) < 1),
-        (1/(pi*s*(1 - 1/s**2)), Abs(s**(-2)) < 1), (meijerg(((S(1)/2,), (0, 0)),
-        ((0, S(1)/2), (0,)), polar_lift(s)**2), True)),
-        And(Abs(periodic_argument(polar_lift(s)**2, oo)) < pi,
-        cos(Abs(periodic_argument(polar_lift(s)**2, oo))/2)*sqrt(Abs(s**2)) -
-        1 > 0, Ne(s**2, 1))), (Integral(exp(-s*x)*cosh(x), (x, 0, oo)), True))
+        Piecewise(
+            (   pi*Piecewise(
```

Comment:
I'd want stuff that at's the same level to be indented to the same level, so

```
(subexpression
    (subexpression
```

does not work very well. Issue here is that the opening parentheses belongs at the same level as `subexpression` but `subexpression1` belongs one level further.
What I wouldn't like but find acceptable:

```
(subexpression,
(
    subexpression1,
    subexpression2),
otherstuff)
```

It's looking weird, but I guess everything will look weird.

Another (and usually better) way out is to extract subexpressions out into local variables. It does not solve all such problems but can reduce them. Something like

```
piecewise11 = (-s/(pi*(-s**2 + 1)), Abs(s**2) < 1)
piecewise12 = (1/(pi*s*(1 - 1/s**2)), Abs(s**(-2)) < 1)
piecewise13 = (meijerg(((S(1)/2,), (0, 0)), ((0, S(1)/2), (0,)), polar_lift(s)**2), True)
condition11 = Abs(periodic_argument(polar_lift(s)**2, oo)) < pi
condition12 = cos(Abs(periodic_argument(polar_lift(s)**2, oo))/2)*sqrt(Abs(s**2)) - 1 > 0,
condition13 = Ne(s**2, 1)
piecewise1 = ((piecewise11, piecewise12, piecewise13), condition1)
condition1 = And(condition11, condition12, condition13)
piecewise2 = (Integral(exp(-s*x)*cosh(x), (x, 0, oo)), True)
assert Piecewise((piecewise1, condition1), piecewise2)
```

Just to demonstrate the general approach, I think there are some typos in that, and I'm not 100% content with the naming yet.